packages/active-record/src/-private/builders/find-record.ts

```diff
@@ -99,7 +99,7 @@ export function findRecord(
 
   const url = buildBaseURL(urlOptions);
   const headers = new Headers();
-  headers.append('Content-Type', 'application/json; charset=utf-8');
+  headers.append('Accept', 'application/json; charset=utf-8');
```

Comment:
I removed `Content-Type` from the builders because we want the out-of-the-box CORS experience to be great, and CORS only allows `Content-Type` to be `application/x-www-form-urlencoded` `multipart/form-data` or `text/plain` before it will opt for a preflight request.